### ISSUE TITLE
 error 400

### ISSUE THREAD

**Issue by manhoos69** [moom0o/PhotoSphereStudio#11]
guys please help me idk what to do
when i upload photo it appear 
my photos are 360 i sure.
![F46119DE-1017-47CD-9B10-0B724E6C9D71](https://github.com/moom0o/PhotoSphereStudio/assets/82904968/a58b9fb5-a9e4-4c59-9d26-2f30cda8880f)

**Comment by alexiscoutinho** [comment]
How did you take the photos?

**Comment by manhoos69** [comment]
> How did you take the photos?

i used an app named HDReye on iPhone it takes 360 photos i sure they are 360.

**Comment by alexiscoutinho** [comment]
Could you send it here?

**Comment by manhoos69** [comment]
> Could you send it here?

![F84942A4-347B-402E-83D3-4B16C407BEE9](https://github.com/moom0o/PhotoSphereStudio/assets/82904968/daa2b5e8-fc3b-476f-9921-9d09a14ec401)
 
it is 9mb dont worry about resolution and quality

**Comment by alexiscoutinho** [comment]
I wanted to try uploading it with another program to test if the problem wasn't with the image, so I needed the original photo. Alternatively, you could check the photo's metadata with [exiftool](https://exiftool.org/). It should contain metadata like this:
```
Use Panorama Viewer             : True
Projection Type                 : equirectangular
Cropped Area Image Height Pixels: 4682
Cropped Area Image Width Pixels : 10752
Full Pano Height Pixels         : 5376
Full Pano Width Pixels          : 10752
Cropped Area Top Pixels         : 0
Cropped Area Left Pixels        : 0
First Photo Date                : 2023:03:01 17:18:17.548Z
Last Photo Date                 : 2023:03:01 17:19:50.3Z
Source Photos Count             : 30
Pose Heading Degrees            : 8.0
````

**Comment by moom0o** [comment]
@manhoos69 Try downloading exiftool and running `.\exiftool.exe -UsePanoramaViewer=true -ProjectionType=equirectangular FILE.jpeg`

**Comment by ckoerner** [comment]
I'm getting the same error on an image. Initially I tried uploading the largest jpeg I could (104.6 MB). Thinking it might be a filesize issue (even though that's not reported in the error message) I tried smaller sizes. A 50% version (25.7), and finally a 25% at 6.5 MB.

exiftool reports the following for all images (metadata is kept intact across resizing). Here is the report for the full size image:

Use Panorama Viewer             : True
Cropped Area Image Height Pixels: 9355
Cropped Area Image Width Pixels : 18710
Full Pano Width Pixels          : 18710
Full Pano Height Pixels         : 9355
Projection Type                 : equirectangular

Attached is a 25% version of the image:

![DJI_0001_DJI_0026-edit-25](https://github.com/user-attachments/assets/72cd9259-dde7-4220-a12b-5eb90cd60983)
